pwiz_tools/Skyline/Test/CommandLineAssayImportTest.cs

```diff
@@ -268,7 +269,9 @@ private static void ValidateIrtAndLibrary(SrmDocument docAfter)
                     foreach (var nodeTran in nodePep.TransitionGroups.SelectMany(g => g.Transitions))
                     {
                         if (!nodeTran.HasLibInfo)
-                            Assert.Fail("Missing library info from {0} - {1}", nodePep, nodeTran.GetDisplayText(new DisplaySettings(nodePep, false, 0, 0)));
+                            Assert.Fail("Missing library info from {0} - {1}", nodePep, nodeTran.GetDisplayText(new DisplaySettings(

```

Comment:
Seems overly complicated new code compared with supplying nodePep and false, 0, 0